changelog/@unreleased/pr-4346.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: feature
+break:
```

Comment:
wat

Reply:
good catch, this will break autorelease, you want this to be `feature`.

Reply:
The fix was also wrong, new one should be the final.